.travis.yml

```diff
@@ -1,38 +1,38 @@
 language: php
 
-sudo: false
-
-branches:
-  only:
-    - master
-
 cache:
   directories:
     - $HOME/.composer/cache/files
 
 matrix:
   include:
     - php: 5.3
+      dist: precise
     - php: 5.4
+      dist: trusty
     - php: 5.5
+      dist: trusty
     - php: 5.6
-    - php: hhvm
-    - php: nightly
     - php: 7.0
-      env: COVERAGE=yes
-    - php: 7.0
-      env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
+    - php: 7.1
+    - php: 7.2
+    - php: 7.3
+      env:
+        - COVERAGE=yes
+        - ANALYSIS=yes
+    - php: 7.4
```

Comment:
I think tests should be performed on PHP 8 as well.

Reply:
nightly is php 8 but it exists probably a better tag now

Reply:
As an alternative, tests could be moved to GitHub actions. See https://github.com/yiisoft/strings/tree/master/.github/workflows as an example. There, you'll have PHP 8 beta.

Reply:
I saw many projects doing this, this month. We are on GA too in my projects. But I think we must first resolve PHP8 installation/CI and then refacto the CI process

Reply:
We can now use `8.0` in Travis.

Reply:
Travis isn't giving enough credits to OpenSource anymore so a switch to GitHub actions is inevitable.

Reply:
This could be done separately, it's not related to this PR.

Reply:
Then it's ready to merge except https://github.com/webmozart/glob/pull/19/files#r511509629 should be applied.

Reply:
Ok then I will make the modif